assets/src/scripts/monitoring-location/components/map/nldi-mapping.js

```diff
@@ -77,21 +51,38 @@ export const addNldiLayers = function (map, upstreamFlows, downstreamFlows, upst
       });
     };
 
-    const fetchNldiLinesLayer = function(nldiData, style) {
+    const getNldiLinesLayer = function(nldiData, style) {
```

Comment:
I changed the names here from "fetch*" to "get" because we aren't actually making a web service call and our convention is to use "fetch" for that. I think this makes it clearer that it's just transforming the data into a layer object.